### DISCUSSION TITLE
Why the input bar height is changed after the element below turn into scroll mode?

### DISCUSSION THREAD

**Discussion by OriLiMu** [ScratchAddons/ScratchAddons#7659] [Ideas]
Here is my [code](https://github.com/woshichuanqilz/ScratchAddons/tree/1e67aeb3a5cc4dc196f7305b6103ca7ae4f5f3bf)
Code path addons/find-references. 

The issue is I get a input on the top, and a ul element below. 
When the ul element turn to **scroll** view(related to the css setting `overflow: scroll`). The height of the input is shrink. 

I don't know why here.

The step to reproduce the issue: 
1. click on the eye button on the right to call out the search panel. 
2. search for the variable name let say "VarTmp"(with 5 more used that each usage will be put in the list below, 5 more li item will make the list go into `scroll` mode).
3. click on the list item below you will find that the height of the input bar is shrink
4. Notice that the list must go into the scroll mode, or the issue will not happens.

# Image
Normal height of the input bar should be like this: 
![image](https://github.com/user-attachments/assets/9c188290-762e-4c53-8239-9434eb82424e)
Shrinked height. 
![image](https://github.com/user-attachments/assets/5a3868cc-22e9-42ad-8050-31600b44ba8c)

# About the code

There is a element with class `class="sa-fr-dropdown-out"`. When you click on the input, this element will add an `visible` class name. If the input height is already shrink, this method can bring the normal height back. 

You can use the id **floatwindow** to location the whole floatwindow

**Comment by Samq64** [comment]
Could you open a draft PR?

**Reply by OriLiMu** [reply]
Hi I make some mistake when make the pr. 
The right steps should be `git checkout -b BRANCH-NAME upstream/master` first. Then change the code. 
But I change the code first without checkout the upstream/master.
I ask gpt and do the following steps:
```

git checkout -b fr-branch
git checkout master
git remote add upstream git@github.com:ScratchAddons/ScratchAddons.git
git fetch upstream
git reset --hard upstream/master
git push origin fr-branch
```
I should use fr-branch to make the pr. 
But the fr-branch cant be find here . So any way to fix it? 
![image](https://github.com/user-attachments/assets/6e3fcece-b069-471b-862c-0742e9741755)

And where is this button? 
![](https://github.com/github/docs/raw/78aee663ffc3f1f3eb1668efa0387e0febc97ede/assets/images/help/pull_requests/pull-request-compare-pull-request.png?raw=true)

**Reply by WorldLanguages** [reply]
Here is a direct link so that you can create the pull request: https://github.com/ScratchAddons/ScratchAddons/compare/master...woshichuanqilz:ScratchAddons:fr-branch

It will say there are conflicts, but you can create the PR anyway.
Please check the "allow access to secrets by other contributors" box when creating the PR.

**Reply by OriLiMu** [reply]
> Here is a direct link so that you can create the pull request: [master...woshichuanqilz:ScratchAddons:fr-branch](https://github.com/ScratchAddons/ScratchAddons/compare/master...woshichuanqilz:ScratchAddons:fr-branch)
> 
> It will say there are conflicts, but you can create the PR anyway. Please check the "allow access to secrets by other contributors" box when creating the PR.

 
What should I do if I change the code another time. Just make the pr follow the basic step [here](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CONTRIBUTING.md#opening-a-pull-request) is enough?

**Reply by Samq64** [reply]
You can keep pusing changes after the PR is submitted but if you want to work on another then start with the creating a branch step.

**Reply by OriLiMu** [reply]
> You can keep pusing changes after the PR is submitted but if you want to work on another then start with the creating a branch step.

Can you help me solve  the issue I mentioned here. About the shrinked height of the input bar

**Reply by WorldLanguages** [reply]
> What should I do if I change the code another time

Any new commits in the branch will update the respective pull request automatically.
